package.json

```diff
@@ -44,24 +44,24 @@
     "url": "https://jira.ixsystems.com"
   },
   "engines": {
-    "node": ">=18.19.1"
+    "node": ">=22.0.0"
```

Comment:
We need to confirm that this is actually required by Angular and is supported by our build pipeline.